tests/test_aggr_regressions.py

```diff
@@ -94,6 +95,38 @@ class X(dj.Lookup):
     contents = zip(range(10))
 
 
+@schema
+class Parent(dj.Manual):
+    definition = """
+    # Parent table
+    id: int
+    ---
+    txt = "NA" : varchar(32)
+    """
+
+
+@schema
+class Child(dj.Computed):
+    definition = """
+    # Child table
+    -> Parent
+    start_time: datetime(6)
+    ---
+    timestamps: longblob
+    """
+
+    def make(self, key):
+        t = np.random.poisson(1000, 10)
+        t.sort()
+        self.insert1(
+            {
+                "start_time": datetime.datetime.now(),
+                "timestamps": t,
+                **key,
+            }
+        )
+        print(f"ingested w/ {key}")
+
```

Comment:
@jverswijver Think we can possibly repurpose the following tables for what these were used for:
- `tests/schema_uuid.py:Topic`
- `tests/schema_uuid.py:Item`

If not, then it would just be best to place these table a bit higher up so they could be reused in the future.